demo/Google.Pubsub.V1.Demo/DemoCommands.CreateSubscription.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using System;
+
+namespace Google.Pubsub.V1.Demo
+{
+    public partial class DemoCommands
+    {
+        /// <summary>
+        /// Creates a pubsub subscription to a topic in a Google Cloud Platform project.
+        /// It is assumed that the topic exists before this is called.
+        /// </summary>
+        public static void CreateSubscription(string projectId, string subscriptionId, string topicId)
+        {
+            // This creates a client (actually an instance of SubscriberClientImpl) synchronously, using the
+            // application default credentials, and connecting to the default endpoint. Optional parameters allow
+            // an alternative endpoint to be specified, as well as general client settings and credentials. Most
+            // applications can simply omit these.
+            // This method completes synchronously; use CreateAsync for an asynchronous equivalent.
+            var client = SubscriberClient.Create();
```

Comment:
The async part is different and understandable since there isn't such a thing as an async constructor...